aws_lambda.py

```diff
@@ -18,7 +18,17 @@
     env_option = {}
     if SENTRY_ENV:
         env_option = {"environment": SENTRY_ENV}
-    sentry_sdk.init(SENTRY_DSN, integrations=[AwsLambdaIntegration()], **env_option)
+    if os.getenv("AWS_LAMBDA_FUNCTION_NAME"):
+        # We're running in AWS. See https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html
+        from sentry_sdk.integrations.aws_lambda import AwsLambdaIntegration
+        integrations = [AwsLambdaIntegration()]
+    elif os.getenv("FUNCTION_TARGET", os.getenv("GOOGLE_CLOUD_PROJECT")):
+        # We're running in Google Cloud. See https://cloud.google.com/functions/docs/configuring/env-var
+        from sentry_sdk.integrations.gcp import GcpIntegration
+        integrations = [GcpIntegration()]
+    else:
+        raise EnvironmentError("Could not determine Cloud environment for Sentry")
```

Comment:
Maybe we should use `RuntimeError` here -- `EnvironmentError` seems to refer to a different idea (see [this SO answer](https://stackoverflow.com/a/69037482))